cli/internal/publish/images/images.go

```diff
@@ -335,6 +335,7 @@ func (f *imageFinder) Transform(node *ast.Document, reader text.Reader, pc parse
 						}
 						if len(el.Title) == 0 { // if no title, make sure the element ends with the link
 							parts := bytes.SplitN(lineLiteral, el.Destination, 2)
+							// nolint:revive
```

Comment:
just curious, what's the issue being suppressed here?

Reply:
more than 5 levels of deep indentation. I'm guessing the linter was upgraded recently on the CLI repo and this is causing linter error.

Reply:
got it, okay - we don't have to fix it here but maybe lets add a comment to explain why it's being suppressed, so some good citizen can refactor it in another PR

Reply:
As in https://github.com/cloudquery/cloudquery/pull/18450